datajunction-server/datajunction_server/models/query.py

```diff
@@ -51,6 +51,17 @@ def __hash__(self):
         return hash((self.name, self.type))
 
 
+class ColumnMetadataDetails(BaseSQLModel):
```

Comment:
Why not just replace the above `ColumnMetadata` or why not simply add the `column` and `node` attributes to `ColumnMetada`?

Reply:
I thought about that, but in some cases it's unnecessary. For example, if we're generating materialization and storing config, we only need the column name and the column type, since that's just consumed internally by DJ and not by users.

Reply:
I see, but these fields could be optional right? Maybe it is better to have a separate class but I don't see it yet.

Reply:
@agorajek on second thought, that's a good point! I moved the fields and made them optional on the `ColumnMetadata` class.